CONTRIBUTING.md

```diff
@@ -1,10 +1,23 @@
 # Přispívání do kódu
 
+* Typické workflow je takové, že projdeš issues tady na githubu, forkneš si repository, připravíš úpravy a pak založíš pull request oproti cityvizor repo (do master větve). Abychom se nepotkali na jedné issue, tak prosím do issue hoď komentář, že na ní začínáš pracovat.
```

Comment:
Jeden bod k diskuzi je právě jestli dělat jen branch nebo si udělat vlastní fork, tedy je dost možné, že se to bude v zápětí měnit.

Reply:
tohle jsme řešili s klukama - můžeme to dát jako default přístup; zároveň pokud nemám admina, tak nemůžu branch pushnout/udělat pull request jako člověk z ulice